int32 robot_id
int32 task_id
---
bool success